### ISSUE TITLE
Navigation needs an aria-label to be distinguished from other nav landmarks

### ISSUE THREAD

**Issue by afercia** [WordPress/health-check#313]
The

`<nav class="tabs-wrapper">`

needs to be distinguished from other `<nav>` elements in the admin. Needs an aria-label attribute e.g.

`<nav class="tabs-wrapper" aria-label="<?php esc_attr_e( 'Secondary menu' ); ?>">`

See also for core https://core.trac.wordpress.org/changeset/44905 and https://core.trac.wordpress.org/ticket/43398

**Comment by Clorith** [comment]
Sounds good, would it make sense to give it a more descriptive aria label than just "secondary menu", for example "Site health menu"?

**Comment by afercia** [comment]
Given the other `<nav>` elements in core are labelled
```
aria-label="Main menu"
aria-label="Toolbar"
aria-label="Secondary menu"
```

I'd stick with `aria-label="Secondary menu"`.

**Comment by Clorith** [comment]
Sounds like a good plan then, thank you 💯

**Comment by Clorith** [comment]
Fixed in #321
